doc/examples/denoise_patch2self.py

```diff
@@ -68,10 +68,18 @@
 hardi_fname, hardi_bval_fname, hardi_bvec_fname = get_fnames('stanford_hardi')
 data, affine = load_nifti(hardi_fname)
 bvals = np.loadtxt(hardi_bval_fname)
-denoised_arr = patch2self(data, bvals, shift_intensity=True,
-                          clip_negative_vals=False)
+denoised_arr = patch2self(data, bvals, model='ols', shift_intensity=True,
+                          clip_negative_vals=False, b0_denoising=50)
 
 """
+The above parameters should give optimal denoising performance for Patch2Self.
```

Comment:
Optimal in this dataset, or generally? If generally, why are these not the default values? For example, the warning below about ridge hampering performance is rather ominous. Should we change the default to `'ols'`?

Reply:
Done!

Reply:
Optimal in this dataset, or generally?

Reply:
'ols' is optimal generally. Same goes for `shift_intensity=True` and `clip_negative_vals=False` 👍🏽 Ridge does seem to be useful to speedup the regression. I was also looking into using the regression module of `sklearn` and found this comment from the discussion very useful: https://github.com/scikit-learn/scikit-learn/issues/13923#issuecomment-497077229. 

Should I use this? I was thinking of adding an option called `ols_fast` instead of just replacing `ridge`  with a very small `alpha`. What do you think?

Reply:
> 70 tutorials x 5min = 350min =~ 6h more.... so I recommend to pick up not more than 10 tutorials.

Yes! I don't think it's relevant for all tutorials either :)

Here are the ones where I would add it:
1. CSD and MCSD
2. 3D SHORE
3. MAP-MRI
4. DTI
5. DKI
6. FW-DTI
7. WMTI
8. Basic Tracking EuDX
9. Deterministic Tracking
10. Probabilistic Tracking 
11. Particle Filter Tracking

Does this sound okay?

Reply:
I think that it's very odd that ridge is faster than OLS, given that they essentially are the same algorithm with a small tweak that shouldn't really affect performance at all. But 🤷‍♂️

Reply:
Yep... me too!😂